self-tests.yml

```diff
@@ -585,6 +596,22 @@ tasks:
             LOCAL_PATH: clients
             EXCLUDE_PATTERN: .cache
             REMOTE_PATH: evergreen/clients/${project}/${revision}
+  - name: swaggo-lint
+    commands:
+      - func: get-project-and-modules
+      - func: get-swaggo
+      - command: shell.exec
+        params:
+          shell: bash
+          working_dir: evergreen
+          script: |
```

Comment:
This script overall seems right, but could we make it consistent with the other linting/verification scripts in `scripts` (like `scripts/verify-client-version-update.sh`)? Putting scripts inline into the YAML (as opposed to making a separate standalone script, or putting it in the Makefile) is slightly less preferred since it makes the YAML quite long and can't be run locally (which can help people check on their own machines if they want to set up their own stuff like git commit hooks for linters).